projects/ifa-dissertation/js/index.js

```diff
@@ -182,48 +178,46 @@ d3.csv("./data/ifa-dissertations.csv", function(error, data) {
       "text-anchor": "middle",
       fill: "white"
     });
+
   function tick(e) {
-    node
-      .each(gravity(.2 * e.alpha))
-      .each(collide(.5))
-      .attr("cx", function (d) {return d.x;})
-      .attr("cy", function (d) {return d.y;});
+    nodes
+      .each(gravity(.1 * e.alpha))
+      .each(collide(0.5))
+      .attr("cx", function (d) { return d.x; })
+      .attr("cy", function (d) { return d.y; });
   }
```

Comment:
Inside the `tick` function which happens periodically on every clock tick (kind of like a draw loop), we set the `cx` and the `cy` of each `<circle></circle>` element after the changes made by `gravity()` and `collide()` functions so that the circles move accordingly on the DOM.